windows/CodePush.Net46/UpdateManager.cs

```diff
@@ -0,0 +1,326 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System.IO;
+using System.IO.Compression;
+using System.Threading.Tasks;
+using System.Runtime.CompilerServices;
+using System;
+#if WINDOWS_UWP
+using Windows.Web.Http;
+#else
```

Comment:
Since we're in the context of the .NET 4.6 project, do we really need these conditional compile directives?